linux_os/guide/system/auditing/auditd_configure_rules/audit_execution_acl_commands/audit_rules_execution_setfacl/rule.yml

```diff
@@ -27,23 +27,25 @@ rationale: |-
 severity: medium
 
 identifiers:
+    cce@rhel8: CCE-88437-9
     cce@sle12: CCE-83189-1
     cce@sle15: CCE-85594-0
 
 references:
     disa: CCI-000130,CCI-000169,CCI-000172,CCI-002884
     nist@sle12: AU-3,AU-3.1,AU-12.1(ii),AU-12(a),AU-12.1(iv),AU-12(c),MA-4(1)(a)
-    srg: SRG-OS-000037-GPOS-00015,SRG-OS-000062-GPOS-00031,SRG-OS-000392-GPOS-00172,SRG-OS-000462-GPOS-00206,SRG-OS-000471-GPOS-00215
+    srg: SRG-OS-000037-GPOS-00015,SRG-OS-000042-GPOS-00020,SRG-OS-000062-GPOS-00031,SRG-OS-000392-GPOS-00172,SRG-OS-000462-GPOS-00206,SRG-OS-000471-GPOS-00215
+    stigid@rhel8: RHEL-08-030330
     stigid@sle12: SLES-12-020610
     stigid@sle15: SLES-15-030430
     stigid@ubuntu2004: UBTU-20-010167
 
 ocil: |-
     To verify that execution of the command is being audited, run the following command:
-    Configure the SUSE operating system to generate an audit record for all uses of the "setfacl" command.
+    Configure the operating system to generate an audit record for all uses of the "setfacl" command.
     Add or update the following rules in the "/etc/audit/audit.rules" file:
-    -a always,exit -F arch=b32 path=/usr/bin/setfacl -F perm=x -F auid>=1000 -F auid!=4294967295 -F key=privileged
-    -a always,exit -F arch=b64 path=/usr/bin/setfacl -F perm=x -F auid>=1000 -F auid!=4294967295 -F key=privileged
+    -a always,exit -F arch=b32 path=/usr/bin/setfacl -F perm=x -F auid&gt;={{{ auid }}} -F auid!=unset -F key=privileged
+    -a always,exit -F arch=b64 path=/usr/bin/setfacl -F perm=x -F auid&gt;={{{ auid }}} -F auid!=unset -F key=privileged
```

Comment:
I just checked the SLE STIGs and they mention `4294967295` instead of `unset`.
I know that other rules selected by SLE profiles that  mention `unset`, but could this raised as a finding?

Reply:
Not a problem as other rules already use this `unset`